service/service.go

```diff
@@ -179,6 +184,7 @@ func (svc *Service) createRouter(ctx context.Context, cfg *config.Config) (route
 
 	// API and Frontend Routers
 	router.Handle("/api/{uri:.*}", apiRouterProxy)
+	router.HandleFunc("/releasecalendar/data", ReleaseData(sc))
```

Comment:
Couldn't we just point the request at the apiRouterProxy? Or does that not work?